expense/src/activities/payment.ts

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export async function payment(id: string): Promise<void> {
```

Comment:
Is there a reason not to include both the activities in the same file?

Reply:
Do you recommend putting them in the same file? I don't have any strong feelings either way, but I think we should be consistent as to whether we recommend separate file per activity or all activities in 1 file.